src/signals/incident-management/containers/IncidentDetail/components/StatusForm/components/ModalHeader/ModalHeader.tsx

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2022 Gemeente Amsterdam
+import type { FC } from 'react'
+import styled from 'styled-components'
+import { Button, Heading, themeColor, themeSpacing } from '@amsterdam/asc-ui'
+import { Close } from '@amsterdam/asc-assets'
+
+export const CloseButton = styled(Button)`
+  border: none;
+  padding: 0;
+  width: 20px;
+  height: 20px;
```

Comment:
These two rules are already covered by the `size` prop on the `CloseButton` component. If not, you can use `size` and `iconSize` props to get the same result.

Reply:
fixed